e2e/actions/l2_challenger_test.go

```diff
@@ -182,6 +208,202 @@ interaction:
 			require.True(t, isValid)
 			txHash = guardian.ActConfirmTransaction(t, big.NewInt(0))
 			includeL1Block(t, miner, guardian.address)
+
+			// check challenger bond amount decreased
+			cBal, err := valPoolContract.BalanceOf(nil, challenger.address)
+			require.NoError(t, err)
+			require.Equal(t, 0, new(big.Int).SetInt64(challengerInitialAmount-1).Cmp(cBal))
```

Comment:
Why checking this in ready to prove status specifically? I think we could check it right after the challenge creation.
(Same in `TestChallengerChallengerTimeout` function.)